src/components/modals/ActionCreatorEditor.tsx

```diff
@@ -315,7 +390,7 @@ const initialState: Readonly<ComponentState> = {
     availableConditionalTags: [],
     expectedEntityTags: [],
     requiredEntityTagsFromPayload: [],
-    requiredEntityTags: [],
+    requiredEntityOrConditionTags: [],
     negativeEntityTags: [],
```

Comment:
Why not the same on negativeEntityTags?

Reply:
Yea, I had left comment saying there is more renaming to do:
https://github.com/microsoft/ConversationLearner-UI/pull/1332#discussion_r333652689